src/engine/strat_engine/backstore/metadata/bda.rs

```diff
@@ -344,31 +307,31 @@ impl StaticHeader {
                         } else if loc_1.initialization_time > loc_2.initialization_time {
                             // If the first header block is newer, overwrite second with
                             // contents of first.
-                            BDA::write(f, &buf_loc_1, MetadataLocation::Second)?;
+                            loc_1.write(f, MetadataLocation::Second)?;
```

Comment:
I wondered about one thing: we call `sigblock_from_buf` just above in order to get in here and then I call it again inside `StaticHeader::write`. Should I let it as is or do I rewrite the whole thing differently? (not sure if it's really necessary but asking just in case...)